slideview/src/main/java/ng/max/slideview/SlideView.java

```diff
@@ -33,6 +33,7 @@
     protected ColorStateList slideBackgroundColor;
     protected ColorStateList buttonBackgroundColor;
     protected boolean animateSlideText;
+    protected boolean mReverseSlide;
```

Comment:
Let's change this to `reversed`. Hungarian notations are not used in the project.